Sources/SwiftFusion/Geometry/Rot3.swift

```diff
@@ -130,18 +109,39 @@ public extension Matrix3Coordinate {
       pullback
     )
   }
+}
+
+extension Matrix3Coordinate: LieGroupCoordinate {
+  /// Creates the group identity.
+  public init() {
+    self.init(
+      1, 0, 0,
+      0, 1, 0,
+      0, 0, 1
+    )
+  }
   
   /// Product of two rotations.
   @differentiable(wrt: (lhs, rhs))
-  static func * (lhs: Matrix3Coordinate, rhs: Matrix3Coordinate) -> Matrix3Coordinate {
+  public static func * (lhs: Matrix3Coordinate, rhs: Matrix3Coordinate) -> Matrix3Coordinate {
     Matrix3Coordinate(matmul(lhs.R, rhs.R))
   }
   
   /// Inverse of the rotation.
   @differentiable
-  func inverse() -> Matrix3Coordinate {
+  public func inverse() -> Matrix3Coordinate {
     Matrix3Coordinate(R.transposed())
   }
+
+  @differentiable(wrt: v)
+  public func Adjoint(_ v: Vector3) -> Vector3 {
+    return Vector3(matmul(R, v.tensor.reshaped(to: [3, 1])).reshaped(to: [3]))
```

Comment:
Done.

About further optimizations, a huge improvement will come from https://github.com/borglab/SwiftFusion/issues/52, and we shouldn't spend effort optimizing Rot3 before doing that because that will change the whole implementation a lot.